dirs/dirs.go

```diff
@@ -126,9 +128,11 @@ func SetRootDir(rootdir string) {
 
 	switch release.ReleaseInfo.ID {
 	case "fedora", "centos", "rhel", "arch":
-		SnapMountDir = filepath.Join(rootdir, "/var/lib/snapd/snap")
+		BaseSnapMountDir = "/var/lib/snapd/snap"
```

Comment:
That makes it Raw, except other things raw are not Raw. Still pretty misleading and error prone. Not to mention "raw" doesn't really say much in this context.

See the suggestion in the other point.